### PR TITLE
Changes for Notification for response submitted

### PR DESCRIPTION
**Pull request by khurshedk**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the enable_keep_helm label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/common/task-list/components/notification-banner/content.ts [hunk 1]
```diff
@@ -349,6 +349,60 @@ const en: NotificationBannerContent = {
...
+            {
+              contents: [
+                {
+                  text: '{nameOfRespondentAp13Eng} has responded to your application',
+                },
+              ],
+              links: [
+                {
+                  text: 'View the response',
+                  href: applyParms(VIEW_ALL_ORDERS, { partyType: PartyType.APPLICANT }),
```
Comment (yogendraU) [inline src/main/steps/common/task-list/components/notification-banner/content.ts:364]: This should be redirected to View All documents once its ready for integration

File: src/main/steps/common/task-list/components/notification-banner/index.ts [hunk 2]
```diff
@@ -45,19 +45,34 @@ export const getNotificationBannerConfig = (
     notificationConfig[caseType].hasOwnProperty(partyType)
     ? notificationConfig[caseType][partyType]
         .map(config => {
-          const { id, show } = config;
-
+          const { id, show, respondentNames} = config;
+          console.log(respondentNames);
```
Comment (yogendraU) [inline src/main/steps/common/task-list/components/notification-banner/index.ts:49]: remove console statements

Comment (khurshedk) [reply src/main/steps/common/task-list/components/notification-banner/index.ts]: removed console logs

**Comment by channa-patil** [comment]
Changes are taken care as part of https://github.com/hmcts/prl-citizen-frontend/pull/1623
Closing this.

**Pull request closed without merging**